ament_tools/build_types/cmake.py

```diff
@@ -284,7 +284,7 @@ def _make_test(self, context, build_type, prefix):
             return BuildAction(cmd)
         else:
             self.warn("Could not run tests for '{0}' package because it has no "
-                      "'test' target".format(build_type))
+                      "'test' target".format(context.package_manifest.name))
```

Comment:
Same comment as https://github.com/ament/ament_tools/pull/164#discussion_r141753538

Reply:
Rebased and should be fixed in 64bac93